src/Contact/src/Repository/MessageRepository.php

```diff
@@ -4,20 +4,16 @@
 
 namespace Frontend\Contact\Repository;
 
-use Frontend\Contact\Entity\Message;
 use Doctrine\ORM\EntityRepository;
+use Dot\AnnotatedServices\Annotation\Entity;
+use Frontend\Contact\Entity\Message;
 
 /**
- * Class MessageRepository
- * @package Frontend\Contact\Repository
+ * @Entity(name="Frontend\Contact\Entity\Message")
  * @extends EntityRepository<object>
  */
 class MessageRepository extends EntityRepository
```

Comment:
```suggestion
class MessageRepository extends EntityRepository implements MessageRepositoryInterface
```
Without this, MessageRepositoryInterface is not use anywhere.